proposals/PROPOSALS.md

```diff
@@ -0,0 +1,142 @@
+
+The current HF Technical Track members are:
+
+- Emily Pillmore ([@emilypi](https://github.com/emilypi)), Haskell Foundation
+- Richard Eisenberg ([@goldfirere](https://github.com/goldfirere)), Tweag
+- Michael Snoyman ([@snoyberg](https://github.com/snoyberg)), FPComplete
+- Andrew Lelechenko ([@Bodigrim](https://github.com/Bodigrim)), Barclays
+- Davean Scies ([@davean](https://github.com/davean)), XKCD
+- Edward Kmett ([@ekmett](https://github.com/ekmett)), MIRI
+- Theophile Choutri ([@Kleidukos](https://github.com/Kleidukos)), Scrive
+- Gil Mizrahi (@soupi)
+
+### Voting
+
+For a HIP to be accepted, it must fulfill two requirements:
+
+- 66% of the HF Technical Track votes in favor of it
+- The HIP specifies a proposal lead (e.g. an implementor, or project director) who will represent the work done on the project.
+
+### Responsibilities of the members
+
+- Review all HIPs 
+- Play a role in the discussions, learn in advance about the topic if needed, and make up their mind in the voting process.
+
+### Guests
+
+Experts in some fields may be invited to specific meetings as guests when discussing related HIPs. Their input would be important to discuss the current state of the proposal, both its design and implementation.
+
+## Proposal states
+
+The state of a proposal changes over time depending on the phase of the process and the decisions taken by the HF Technical Track . A given proposal can be in one of several states:
+
+1. **Submitted:**  An HF Tech Track member has validated the proposal and checked that meets all the formal requirements. A pull request has been opened against the proposals repo.
+
+2. **Under review:** The proposal will be under review until the next available Tech Track meeting takes place.
+
+3. **Under revision:** Authors are addressing the issues pinpointed throughout the discussion and feedback process from the community and/or the HF Technical Track.
+
+6. **Dormant:** When a HIP has been under revision for more than two iterations (that is, no progress has been made since the last review), it’s considered dormant, in which case any related activity will be paralyzed and the HF Tech Track will not allocate more resources to it.
+
+7. **Postponed:** The HIP has been postponed under some concrete conditions. When these are met, the HIP can be resubmitted.
+
+8. **Rejected:** The HIP has been rejected with a clear and full explanation.
+
+9. **Merged:** The HIP has been accepted and has been merged into the repo.
+
+## How do I submit? ##
+
+The process to submit is simple:
+
+* Submit a Pre-HIP topic to the HF Discourse, soliciting informal feedback.
+* If the feedback is favorable, fork the Haskell Foundation tech-proposals repository, [https://github.com/haskellfoundation/tech-proposals](https://github.com/haskellfoundation/tech-proposals).
+* Create a new HIP file in the `hips/` directory. Use the [HIP template](https://github.com/haskellfoundation/tech-proposals/blob/main/TEMPLATE.md)
```

Comment:
Perhaps add a link back to the PROPOSALS file in the first paragraph of this template? (before the introduction)?

Reply:
Like a self link?